src/game/server/gamecontext.cpp

```diff
@@ -4272,6 +4276,20 @@ void CGameContext::TargetKilled()
 	m_TargetToKillCoolDown = Server()->TickSpeed()*(10 + 3*max(0, 16 - PlayerCounter));
 }
 
+void CGameContext::FlagCollected()
+{
+	int PlayerCount = 0;
+	CPlayerIterator<PLAYERITER_INGAME> Iter(m_apPlayers);
+	while(Iter.Next())
+		PlayerCount++;
```

Comment:
If you want to get the player count without spectators, you can also use Server()->GetActivePlayerCount()

Reply:
I don't see why not 
5c31674